langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockCohereEmbeddingResponse.java

```diff
@@ -0,0 +1,67 @@
+package dev.langchain4j.model.bedrock;
+
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.model.bedrock.internal.BedrockEmbeddingResponse;
+
+import java.util.List;
+
+/**
+ * Bedrock Cohere embedding response
+ */
+class BedrockCohereEmbeddingResponse implements BedrockEmbeddingResponse {
+
+    private String id;
+
+    private List<String> texts;
+
+    private float[] embedding;
+
+    private int inputTextTokenCount;
+
+    @Override
+    public Embedding toEmbedding() {
+        return new Embedding(embedding);
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public List<String> getTexts() {
+        return texts;
+    }
+
+    public void setTexts(List<String> texts) {
+        this.texts = texts;
+    }
+
+    public void addText(String text) {
+        if (this.texts == null) {
+            this.texts = List.of(text);
```

Comment:
As far as I know

List.of return immutable list

How can we after that add text to immutable?

Reply:
Removed. Anyway it was additonal method. Not needed.